funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -151,50 +115,44 @@ def __init__(
         self.config = config
         log.info(f"Got config: {config}")
 
-        self.client_address = client_address
-        self.interchange_address = interchange_address
-        self.client_ports = client_ports
-        self.suppress_failure = suppress_failure
-
         self.endpoint_dir = endpoint_dir
         self.endpoint_name = endpoint_name
 
         if funcx_client_options is None:
             funcx_client_options = {}
+        # off_process checker is only needed on client side
+        funcx_client_options["use_offprocess_checker"] = False
```

Comment:
Another instance of this. It's the last one I'll comment on. I think all of these changes to set this to false could be in a separate PR. IMO ideally against `main`, but at the very least against the `rabbitmq` branch. It would help de-noise this PR.